nrfx/hal/nrf_nvmc.h

```diff
@@ -133,6 +134,44 @@ NRF_STATIC_INLINE void nrf_nvmc_nonsecure_mode_set(NRF_NVMC_Type *    p_reg,
                                                    nrf_nvmc_ns_mode_t mode);
 #endif
 
+/**
+ * @brief Function for writing a word into the flash
+ *
+ * Before calling this function, the caller must ensure that:
+ *  * The address is word aligned
+ *  * Write mode is enabled and
+ *  * The flash is ready to accept another write
+ *
+ * Users are recommended to use the NVMC driver
+ * nrfx_nvmc_word_write() instead.
+ *
+ * @param[in] addr  Address of the word to write
+ * @param[in] value Value to write
+ */
+NRF_STATIC_INLINE void nrf_nvmc_word_write(uint32_t addr,
+                                           uint32_t value);
+
+/**
+ * @brief Function for reading a word from the flash
+ *
+ * @param[in] addr  Address of the word to read
+ *
+ * @retval Value read from flash
+ */
+NRF_STATIC_INLINE uint32_t nrf_nvmc_word_read(uint32_t addr);
+
+/**
+ * @brief Function for reading a set of words from the flash
```

Comment:
`Function for reading a given number of 32-bit words from the flash into the specified buffer.`